src/cli/src/produce/mod.rs

```diff
@@ -7,7 +7,7 @@ use tracing::debug;
 use fluvio::{Fluvio, TopicProducer};
 use fluvio_types::print_cli_ok;
 use crate::common::FluvioExtensionMetadata;
-use crate::{Result, ConsumerError};
+use crate::error::ConsumerError;
```

Comment:
let's move consumer related under `consumer` module to keep isolated and make it easier to use feature flag